Dockerfile.dapper

```diff
@@ -15,7 +15,7 @@ ENV KUSTOMIZE_VERSION v3.10.0
 # kontainer-driver-metadata branch to be set for specific branch other than dev/master, logic at rancher/rancher/pkg/settings/setting.go
 ENV CATTLE_KDM_BRANCH=dev-v2.6
 
-RUN zypper -n install gcc binutils glibc-devel-static ca-certificates git-core wget curl tar vim less file xz gzip sed gawk iproute2 iptables
+RUN zypper -n install -y gcc binutils glibc-devel-static ca-certificates git-core wget curl tar vim less file xz gzip sed gawk iproute2 iptables jq
```

Comment:
The -y isn't needed, -n already covers this https://www.mankier.com/8/zypper#Global_Options

Reply:
Ah, I put the `-n` after the `install` when I tested this and it didn't work so I added the `-y` 🤦